_dashboard/DotNetPerfMonitor/nuxt.config.ts

```diff
@@ -1,13 +1,23 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
   devtools: { enabled: true },
+  target: 'static',
+  app: {
+	baseURL: '/DotNetPerfMonitor/',
```

Comment:
Fixes problem with URL prefixes when deployed to GitHub Pages

Reply:
This is cool! I tried to add this configuration.... But I directly put target: 'static', and baseURL: '/DotNetPerfMonitor'...  Did not know I had to wrap baseURL key around app object!